pkg/agent/openflow/pipeline.go

```diff
@@ -488,7 +602,7 @@ func (c *client) l3FwdFlowToRemoteViaGW(
 	localGatewayMAC net.HardwareAddr,
 	peerSubnet net.IPNet,
 	category cookie.Category) binding.Flow {
-	l3FwdTable := c.pipeline[l3ForwardingTable]
+	l3FwdTable := c.pipeline[localPodL3ForwardingTable]
```

Comment:
why this is in localPodL3ForwardingTable? I think in the discussion it's L3ForwardingTable

Reply:
Yes, it should be like you said. Updated.